runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -260,6 +260,29 @@ public void removeNode(@Nonnull Layout currentLayout,
         reconfigureSequencerServers(currentLayout, newLayout, false);
     }
 
+    /**
+     * Attempts to force commit a new layout to the cluster.
+     *
+     * @param currentLayout the current layout
+     * @param forceLayout the new layout to force
+     * @throws QuorumUnreachableException
+     */
+    public void forceLayout(@Nonnull Layout currentLayout, @Nonnull Layout forceLayout) {
+        currentLayout.setRuntime(runtime);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.QuorumUnreachableException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)